package.json

```diff
@@ -161,70 +161,83 @@
       "default": "pdf",
       "order": 10
     },
+    "producer": {
+      "description": "Program to use when post-process DVI output in order to produce PDF. This is not used for pdflatex based builders. Currently only supported for latexmk.",
+      "type": "string",
+      "enum": [
+        "default",
```

Comment:
Default was meant to protect the user's `.latexmkrc` file in case they had a setting in there. I'll look at the latexmk code and see what order the evaluation happens in and whether this is really needed.